cred-manager/prisma-lite/schema.prisma

```diff
@@ -137,3 +137,9 @@ model Update {
 
   @@map("updates")
 }
+
+model AnonymousUsers {
+  id         String   @id @default(uuid())
+  email      String
```

Comment:
I think we need to make the email unique?
I just tested it deleting the cache and going through again.

so if the email is already registered, we just go through to the github login component, in that sense it would still return success from the API but it would not register it again as duplicated on the database